salat-core/src/test/scala/com/novus/salat/test/BinarySpec.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2010 - 2012 Novus Partners, Inc. (http://www.novus.com)
```

Comment:
Yeah, there are odds and ends like this (check out the copyright for Casbah sometime, btw).

The copyright is as follows:

```Copyright Novus 2011 - May 2015, copyright to Rose Toomey June 2015 - present, copyright to individual contributors where noted.```